README.md

```diff
@@ -2,7 +2,11 @@
 
 [![GitHub Actions](https://github.com/larq/zoo/workflows/Unittest/badge.svg)](https://github.com/larq/zoo/actions?workflow=Unittest) [![Codecov](https://img.shields.io/codecov/c/github/larq/zoo)](https://codecov.io/github/larq/zoo?branch=master) [![PyPI - Python Version](https://img.shields.io/pypi/pyversions/larq-zoo.svg)](https://pypi.org/project/larq-zoo/) [![PyPI](https://img.shields.io/pypi/v/larq-zoo.svg)](https://pypi.org/project/larq-zoo/) [![Join the community on Spectrum](https://withspectrum.github.io/badge/badge.svg)](https://spectrum.chat/larq) [![PyPI - License](https://img.shields.io/pypi/l/larq-zoo.svg)](https://github.com/plumerai/larq-zoo/blob/master/LICENSE) [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
 
-Larq Zoo contains reference implementations of popular Binarized Neural Networks.
+Larq Zoo consists of a `literature` and a `sota` submodule.
+
+The `literature` submodule contains replications from research papers (all current models). These models are intended to provide a stable reference for ideas presented in specific papers. The model implementations will be maintained, but we will not attempt to improve these models over time by applying new training strategies or architecture innovations.
```

Comment:
Small thing but I prefer having `.md` files formatted with each sentence on its own line (over a whole paragraph on one line or hard-wrapping after `n` characters), which makes it easier to keep track of changes in git.